bears/elm/ElmLintBear.py

```diff
@@ -0,0 +1,37 @@
+from shutil import which
+
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='elm-format',
+        use_stdin=True,
+        use_stderr=True,
```

Comment:
alright :)
if you don't need `stdout` then, you can disable it by `use_stdout = False`  :+1: